PWGHF/Tasks/taskMcEfficiencyToXiPi.cxx

```diff
@@ -0,0 +1,435 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file taskMcEfficiencyToXiPi.cxx
+/// \brief Task to analyse the MC efficiency for toXiPi charm baryon decay ("found / trackable")
+///
+/// \author Federica Zanone, Heidelberg University
+
+#include "CommonConstants/PhysicsConstants.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/HistogramRegistry.h"
+#include "Framework/O2DatabasePDGPlugin.h"
+#include "Framework/runDataProcessing.h"
+
+#include "Common/Core/RecoDecay.h"
+#include "Common/DataModel/TrackSelectionTables.h"
+
+#include "PWGHF/Core/HfHelper.h"
+#include "PWGHF/DataModel/CandidateReconstructionTables.h"
+#include "PWGHF/DataModel/CandidateSelectionTables.h"
+
+using namespace o2;
+using namespace o2::analysis;
+using namespace o2::constants::physics;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+
+struct HfTaskMcEfficiencyToXiPi {
+
+  Configurable<double> rapidityCharmBaryonMax{"rapidityCharmBaryonMax", 0.5, "Max absolute value of rapidity for charm baryon"};
+  Configurable<double> mcLfAcceptanceEta{"mcLfAcceptanceEta", 1.0, "Max absolute value of eta for LF daughters"};
+  Configurable<double> mcPionFromCascadeAcceptancePt{"mcPionFromCascadeAcceptancePt", 0.2, "Min value of pt for pion <-- cascade"};
+  Configurable<double> mcPionFromCharmBaryonAcceptanceEta{"mcPionFromCharmBaryonAcceptanceEta", 0.8, "Max absolute value of eta for pion <-- charm baryon"};
+  Configurable<double> mcPionFromCharmBaryonAcceptancePt{"mcPionFromCharmBaryonAcceptancePt", 0.5, "Min value of pt for pion <-- charm baryon"};
```

Comment:
Start the name with what quantity the object represents. I don't think the `mc` prefixes are needed.